Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py

```diff
@@ -275,7 +275,7 @@
         'Geometry/EcalSimData/data/ESProdCuts.xml',
         'Geometry/TrackerSimData/data/PhaseI/trackerProdCuts.xml',
         'Geometry/TrackerSimData/data/trackerProdCutsBEAM.xml',
-        'Geometry/MuonSimData/data/2021/muonProdCuts.xml',
+        'Geometry/MuonSimData/data/2021/v1/muonProdCuts.xml',
```

Comment:
@boudoul - shouldn't be ```Geometry/MuonSimData/data/muonProdCuts/2021/v1/muonProdCuts.xml```?

Reply:
Please, see rule No5 in https://github.com/cms-sw/cms-sw.github.io/blob/master/cms_coding_rules.md

Reply:
hi @ianna .. mmh you are suggesting to add yet another subdirectory to https://github.com/cms-sw/cmssw/tree/master/Geometry/MuonSimData/data  ?   Geometry/MuonSimData/data/muonProdCuts doesn't exist ...

Reply:
sorry, it's not me, the coding rules do :-)